templates/part.eventsattendees.php

```diff
@@ -52,18 +52,16 @@
 					id="attendeecutype_{{$id}}">
 				</select>
 				<div class="attendeeopt pull-right">
-					<input
-						type="checkbox" class="attendeecheckbox event-checkbox"
+					<input type="checkbox" name="attendeecheckbox" class="checkbox"
```

Comment:
All of the changes to this file should be made in a separate PR, not in this one.